src/pint/simulation.py

```diff
@@ -1,7 +1,10 @@
 """Functions related to simulating TOAs and models
 """
+from __future__ import annotations
```

Comment:
This is only relevant if there are forward references, as far as I can tell, and I couldn't spot any. Is this actually needed?